### ISSUE TITLE
[Conversation] - Unable to intialise Context variables 

### ISSUE THREAD

**Issue by DavidJFowler** [watson-developer-cloud/dotnet-standard-sdk#170]
Prior to version 1.3.0 the Context property of the MessageRequest class was dynamic. This made it possible to initialise variables when creating a new context:
```
var context = new
{
    myVariable = "Hello world"
}

var request = new MessageRequest { Input = new InputData { Text = "Hello" }, Context = context };
```

In version 1.3.0, Context is strongly typed so this is no longer possible.

**Comment by mediumTaj** [comment]
I will look into this.

**Comment by mediumTaj** [comment]
Can you check if the `hotfix-dynamic-context` branch works as expected for you? If it works, I will release a patch and look at the issue further in the generator.

**Comment by DavidJFowler** [comment]
Hi,

Yes, that looks good.

Thanks

**Comment by mediumTaj** [comment]
Fixed in `v1.3.1`. Deploying to NuGet as soon as CI passes. 
https://github.com/watson-developer-cloud/dotnet-standard-sdk/releases
